pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml

```diff
@@ -82,8 +68,362 @@
     <bundle start-level="90">blueprint:mvn:pentaho/pentaho-blueprint-activators/${project.version}/xml/camel-jms
     </bundle>
   </feature>
+  
+    <!-- Spring 2 support -->
+    <feature name="spring" description="Spring 2.x support" version="2.5.6.SEC03" resolver="(obr)">
```

Comment:
I searched but did not fine a public spring feature for 3.0.3. That is when I copied everything to our standards file